gaj23/rancid
# Rlt/feature/format money

## *Purpose:*
*What does this merge do? Check all that apply.*


## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- displays film class when a movie is selected 
- returns to home screen when home icon is clicked

## *Instructions for reviewer:*
*Where should the reviewer start?*
- Film.js

*How should this be manually tested?*
- As a user, one should be able to click on a movie and have it display a movies details. 
- As a user, when one is viewing a single movie's details, you should be able to click on the home icon to see the home screen with all movies. 

*Any background context you want to provide?*
*What are the relevant tickets?*
closes #10 closes #9 closes #4 closes #8 

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
- page is currently styled for mobile use only. Desktop is usable but not up to par. 


## Checklist:
